packages/hydrooj/src/model/problem.ts

```diff
@@ -491,8 +492,9 @@ export class ProblemModel {
                 }
                 const overrideContent = findOverrideContent(path.join(tmpdir, i));
                 if (pdoc.difficulty && !Number.isSafeInteger(pdoc.difficulty)) delete pdoc.difficulty;
+                const title = typeof pdoc.title === 'string' ? pdoc.title.trim() : (pdoc.title as any).toString().trim();
```

Comment:
Title should ALWAYS be a string. You can add validation to those fields but please do not introduce extra complexity to the problem format itself.